packages/client/hmi-client/src/types/Types.ts

```diff
@@ -1148,6 +1157,15 @@ export enum EvaluationScenarioStatus {
     Stopped = "STOPPED",
 }
 
+export enum TaskStatus {
```

Comment:
well.... similar fields.... but worth combining

Reply:
Made an issue for https://github.com/DARPA-ASKEM/terarium/issues/2627 . Doing it properly so that tasks are guarded against invalid states to prevent race conditions will take time